src/pydocstyle/violations.py

```diff
@@ -82,10 +94,10 @@ def __str__(self):
                                ['filename', 'line', 'definition', 'message',
                                 'explanation', 'lines']})
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return str(self)
 
-    def __lt__(self, other):
+    def __lt__(self, other: 'Error') -> bool:
```

Comment:
Why is Error a string?

Reply:
![image](https://user-images.githubusercontent.com/80665/47381027-88859980-d707-11e8-92b0-d89838ab4003.png)